docs/admin/common_tasks.md

```diff
@@ -0,0 +1,117 @@
+
+## Single host setup with PMEM and NVMe
+
+1. Check PMEM and NVMe are discovered by the system. Format and reset them.
+2. Check network configuration. Check that `ib` interfaces are active.
+3. Install `daos-server` and `daos-client` RPM.
+4. Generate certificate files.
+5. Place one of the example configs from `utils/config/examples` to
+`/etc/daos` and adjust it based on the environment. e.g., `access_points`,
+`bdev_class`.
+6. Start `daos_server`.
+7. Use `dmg config generate` to generate the config file that contains PMEM and
+NVMe.
+8. Define the certificate files in the server config.
+9. Start server with the generated config file.
+10. Check that it's waiting for SCM format. Call dmg storage format.
+11. Create a small pool; ~500MB.
+12. Define the certificate files in the agent config.
+13. Start agent.
+14. Create a POSIX container with daos command.
+15. Mount the container with dfuse.
+16. Add a large file that's less than the 500MB pool size into the container.
+17. Call `dmg pool query` and check that the free size has declined.
+
+## Multiple host setup with PMEM and NVMe
+
+1. Check PMEM and NVMe configurations are homogeneous. i.e., same number of
+disks, size, address, etc.
+2. Check network configuration. Check that both the server and the client hosts
+can communicate with the Infiniband interface.
+3. Install the same version of `daos-server` and `daos-client` RPM to all the
+hosts.
+4. Generate certificate files and distribute them to all the hosts.
+5. Place one of the example configs from `utils/config/examples` to
+`/etc/daos` of one of the server hosts and adjust it based on the environment.
+e.g., `access_points`, `bdev_class`.
+6. Start `daos_server`.
+7. Use dmg config generate to generate the config file that contains PMEM and
+NVMe.
+8. Distribute the config file to `/etc/daos` of all hosts.
+9. Start server on all the hosts.
+10. Check that it's waiting for SCM format. Call dmg storage format against all
+server hosts.
+11. Check that the servers are running on all the hosts with `dmg system query
+--verbose`.
+12. Create a small pool; ~500MB.
+13. Define the certificate files in the agent config in the client host.
+14. Start agent.
+15. Create a POSIX container with daos command.
+16. Mount the container with dfuse.
+17. Add a large file that's less than the 500MB pool size into the container.
+18. Call dmg pool query and check that the free size has declined.
+
+## Pool size management
+
+1. Start DAOS server with PMEM + NVMe and format.
+2. Call `dmg storage scan --verbose` to get the available size allocated to this
+server. Find the NVMe size.
+3. Create a pool with the size half of the NVMe with `--size`. For example, if
+the total NVMe size allocated is 1.6TB, call
```

Comment:
Very recent is now `--size 50%`: https://github.com/daos-stack/daos/commit/81671fd6bf223bb8064ff54e6a757f38b232183d

Reply:
Updated the section to introduce the use of percentage.